styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -120,12 +121,14 @@
   public static final String STYX_MODE_DEVELOPMENT = "development";
 
   public static final int SCHEDULER_TICK_INTERVAL_SECONDS = 2;
+  public static final int SUBMISSION_RATE_LIMITER_UPDATE_TICK_INTERVAL_SECONDS = 5;
```

Comment:
We now have a number of inconsistent ad-hoc "get config value from storage" implementations. Cf. https://github.com/spotify/styx/pull/75/files#diff-a473be0c753046d2e7f56a00c3ed5b62L127

I wonder if it would be valuable to un-blaze that trail, and do "CONFIG_UPDATE_INTERVAL_SECONDS" or something, with consistent (such that doesn't drown in warning.log noise) error signaling etc.